data/bus-times.json

```diff
@@ -4,7 +4,7 @@
     "schedules": [
       {
         "days": ["mo", "tu", "we", "th", "fr", "sa"],
-        "stops": ["St. Olaf", "Carleton", "Food Co-op", "Cub/Target", "El Taquila", "Food Co-op", "Carleton", "St. Olaf"],
+        "stops": ["St. Olaf", "Carleton", "Food Co-op", "Cub/Target", "El Tequila", "Food Co-op", "Carleton", "St. Olaf"],
```

Comment:
I will have you know that I copied that straight out of their official map brochure thing. ;P